lib/webpacker/commands.rb

```diff
@@ -5,6 +5,24 @@ def initialize(webpacker)
     @webpacker = webpacker
   end
 
+  def clean
+    return if !config.public_output_path.exist? || !config.public_manifest_path.exist?
+
+    count_to_keep = 2
+    files_in_manifest = manifest.refresh.values.map { |f| File.join config.root_path, "public", f }
+    files_to_be_removed = files_in_manifest.flat_map do |file_in_manifest|
```

Comment:
Might be worth comparing to the logic that Sprockets uses: https://github.com/rails/sprockets/blob/master/lib/sprockets/manifest.rb#L245

Reply:
I've compared the logic and I believe it's fairly identical. They're going about getting the files differently though since they keep old version names in the manifest whereas Webpack does not.